addons/mail/static/src/components/message_in_reply_to_view/message_in_reply_to_view.xml

```diff
@@ -2,22 +2,27 @@
+                <span class="o_MessageInReplyToView_corner position-absolute bottom-0 top-50 pe-4 border-top text-300" t-att-class="{ 'ms-n2': messageInReplyToView.messageView.inDiscuss }" t-attf-class=" {{ messageInReplyToView.messageView.inChatWindow and messageInReplyToView.messageView.message.isCurrentUserOrGuestAuthor ? 'o-align-right border-end' : 'o-align-left border-start' }}"/>
                 <t t-if="!messageInReplyToView.messageView.message.parentMessage.isEmpty">
-                    <b class="o_MessageInReplyToView_author text-muted ms-2">@<t t-esc="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
-                    <span class="o_MessageInReplyToView_body ms-1 o_cursor_pointer" t-on-click="messageInReplyToView.onClickReply">
-                        <t t-if="messageInReplyToView.hasBodyBackLink">
-                            <t t-out="messageInReplyToView.messageView.message.parentMessage.prettyBodyAsMarkup"/>
-                        </t>
-                        <t t-if="messageInReplyToView.hasAttachmentBackLink">
-                            <span class="fst-italic me-2">Click to see the attachments</span>
-                            <i class="fa fa-image"/>
-                        </t>
+                    <span class="o_MessageInReplyToView_wrapOuter d-flex align-items-center text-muted opacity-75 opacity-100-hover o_cursor_pointer" t-attf-class="{{ messageInReplyToView.messageView.inChatWindow and messageInReplyToView.messageView.message.isCurrentUserOrGuestAuthor ? 'pe-3' : 'ps-3' }}" t-on-click="messageInReplyToView.onClickReply">
+                        <img class="o_MessageInReplyToView_avatar me-2 rounded-circle" t-att-src="messageInReplyToView.messageView.message.parentMessage.avatarUrl" t-att-title="messageInReplyToView.messageView.message.parentMessage.authorName" alt="Avatar"/>
+                        <span class="o_MessageInReplyToView_wrapInner overflow-hidden">
+                            <b class="o_MessageInReplyToView_author">@<t t-out="messageInReplyToView.messageView.message.parentMessage.authorName"/></b>
+                            <span class="o_MessageInReplyToView_body text-break">
+                                <t t-if="messageInReplyToView.hasBodyBackLink">
+                                    <t t-out="messageInReplyToView.messageView.message.parentMessage.prettyBodyAsMarkup"/>
+                                </t>
+                                <t t-if="messageInReplyToView.hasAttachmentBackLink">
+                                    <span class="me-2 fst-italic">Click to see the attachments</span>
+                                    <i class="fa fa-image"/>
+                                </t>
+                            </span>
```

Comment:
As said in a prior comment: make a dedicated commit just for change of style for reply